ArduCopter/system.cpp

```diff
@@ -451,6 +451,11 @@ void Copter::allocate_motors(void)
         attitude_control = new AC_AttitudeControl_Multi_6DoF(*ahrs_view, aparm, *motors, scheduler.get_loop_period_s());
         ac_var_info = AC_AttitudeControl_Multi_6DoF::var_info;
 #endif // AP_SCRIPTING_ENABLED
+#if CUSTOMCONTROL_ENABLED == ENABLED && FRAME_CONFIG != HELI_FRAME
+    } else if (CUSTOMCONTROL_ENABLED) {
```

Comment:
This is a little odd, the else if will always be true if it is compiled in. Better to do the else with defines.